src/configuration/resolve_config.rs

```diff
@@ -78,6 +78,7 @@ pub fn resolve_config(config: ConfigKeyMap, global_config: &GlobalConfiguration)
     binary_expression_line_per_expression: get_value(&mut config, "binaryExpression.linePerExpression", false, &mut diagnostics),
     jsx_quote_style: get_value(&mut config, "jsx.quoteStyle", quote_style.to_jsx_quote_style(), &mut diagnostics),
     jsx_multi_line_parens: get_value(&mut config, "jsx.multiLineParens", JsxMultiLineParens::Prefer, &mut diagnostics),
+    jsx_force_content_new_line: get_value(&mut config, "jsx.forceContentNewLine", false, &mut diagnostics),
```

Comment:
A bit verbose, but maybe `jsx.forceNewLineSurroundingContent` would be a bit less ambiguous?